### ISSUE TITLE
Bug in Dev / Test Loss Accumulation?

### ISSUE THREAD

**Issue by ramppdev** [autrainer/autrainer#125]
I stumbled upon the following line:
https://github.com/autrainer/autrainer/blob/8d86aa969d30209ed0c25c11c06d42791bb975d1/autrainer/training/training.py#L861
I am not sure why we have the `+1` here? I traced this back pretty far an it has been there for a long time.

**Comment by ATriantafyllopoulos** [comment]
I think the bug may have been caused because I often use the following snippet to compute the total loss:

```python
loss = 0
for index, (X, Y) in enumerate(loader):
     loss += criterion(model(X), Y)
loss /= (index+1)
```

I can't remember why `(len(loader)+1)` slipped in there, but it's definitely a bug

**Comment by ATriantafyllopoulos** [comment]
Besides logging, does it have any other repercussions?

**Comment by ramppdev** [comment]
It only concerns logging and not actual training, but this carries out to everything that is logged: the metrics csv, plots, ...
